example/src/tests.ts

```diff
@@ -259,8 +259,12 @@ test('can load 1995 conversations from dev network "2k lens convos" account', as
     xmtpClient.address === '0x209fAEc92D9B072f3E03d6115002d6652ef563cd',
     'Address: ' + xmtpClient.address
   )
-
+  let start = Date.now()
   const conversations = await xmtpClient.conversations.list()
+  let end = Date.now()
+  console.log(
+    `Loaded ${conversations.length} conversations in ${end - start}ms`
+  )
```

Comment:
This is consistently giving me ~18s to load 2000 convos.

Reply:
I was kind of thinking we could just leave this log in the tests just so we can kind of see that this number when we run it incase it gets super large etc...

Reply:
I like keeping this as well 👍